src/App.tsx

```diff
@@ -15,36 +17,92 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  SORT_ALPHABET = 'alphabet',
+  SORT_LENGTH = 'length',
+}
+
+const handleSort = (type: SortType | '', isReversed: boolean): string[] => {
```

Comment:
Also, you can move this function to a separate file, import it, and use it inside the component.